### PR TITLE
Pose

### PR DESCRIPTION
**Pull request by vinicius121**
Add sample that uses OpenPose model to track the user joints.

### CODE REVIEW

File: SamplesCS/Samples/FaceDetectionDNN.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    /// Add the files to the bin folder
+    /// </summary>
+    class FaceDetectionDNN : ISample
+    {
+        public void Run()
+        {
+            string finalOutput = "DetectedFaces.jpg";
+            string faceModel = "res10_300x300_ssd_iter_140000_fp16.caffemodel";
+			string configFile = "deploy.prototxt";
+			var image = "faces.jpg";
```
Comment (shimat) [inline SamplesCS/Samples/FaceDetectionDNN.cs]:
```suggestion
            var image = "faces.jpg";
```

File: SamplesCS/Samples/FaceDetectionDNN.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    /// To run this example first download the face model available here:https://github.com/spmallick/learnopencv/tree/master/FaceDetectionComparison/models
+    /// Add the files to the bin folder
+    /// </summary>
+    class FaceDetectionDNN : ISample
+    {
+        public void Run()
+        {
+            string finalOutput = "DetectedFaces.jpg";
+            string faceModel = "res10_300x300_ssd_iter_140000_fp16.caffemodel";
+			string configFile = "deploy.prototxt";
```
Comment (shimat) [inline SamplesCS/Samples/FaceDetectionDNN.cs]:
```suggestion
            string configFile = "deploy.prototxt";
```

File: SamplesCS/Samples/Pose.cs [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+{
+    /// <summary>
+    /// To run this example first download the pose model available here: https://github.com/CMU-Perceptual-Computing-Lab/openpose/tree/master/models
+    /// Add the files to the bin folder
+    /// </summary>
+    class Pose : ISample
+    {
+        public void Run()
+        {
+	    const string Model = "pose_iter_160000.caffemodel";
```
Comment (shimat) [inline SamplesCS/Samples/Pose.cs]: please auto format (fix indentation)

File: SamplesCS/Samples/FaceDetectionDNN.cs [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+			var image = "faces.jpg";
+            // Read sample image
+            var frame = Cv2.ImRead(image);
+            int frameHeight = frame.Rows;
+            int frameWidth = frame.Cols;
+            using (var faceNet = CvDnn.ReadNetFromCaffe(configFile, faceModel)) 
+            {
+				using (Mat blob = CvDnn.BlobFromImage(frame, 1.0, new OpenCvSharp.Size(300, 300), new Scalar(104, 117, 123), false, false))
```
Comment (shimat) [inline SamplesCS/Samples/FaceDetectionDNN.cs]: decrease indent
```c#
using (var faceNet = CvDnn.ReadNetFromCaffe(configFile, faceModel)) 
using (Mat blob = CvDnn.BlobFromImage(frame, 1.0, new OpenCvSharp.Size(300, 300), new Scalar(104, 117, 123), false, false)) 
{
```

File: SamplesCS/Samples/Pose.cs [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+                points.Add((OpenCvSharp.Point)p);
+            }
+            int nPairs = 14; //(POSE_PAIRS).Length / POSE_PAIRS[0].Length;
+
+            for (int n = 0; n < nPairs; n++)
+            {
+                // lookup 2 connected body/hand parts
+                Point2f partA = points[POSE_PAIRS[n][0]];
+                Point2f partB = points[POSE_PAIRS[n][1]];
```
Comment (shimat) [inline SamplesCS/Samples/Pose.cs]:
```suggestion
                Point partB = points[POSE_PAIRS[n][1]];
```

File: SamplesCS/Samples/Pose.cs (hunk 3 above)
Comment (vinicius121) [reply SamplesCS/Samples/Pose.cs]: Can you check it now?

File: SamplesCS/Samples/FaceDetectionDNN.cs (hunk 4 above)
Comment (vinicius121) [reply SamplesCS/Samples/FaceDetectionDNN.cs]: But this is from the Face Detection Sample. Which was already merged.

**Comment by shimat** [comment]
Thanks!

**Pull request merged**